### PR TITLE
Transactions: Fixed the broken link, added additional info

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Fixed the broken link to orders page
Added additional info about fetching transaction details

## Affected documentation pages

https://docs.magento.com/user-guide/sales/transactions.html

### CODE REVIEW

File: src/sales/transactions.md [hunk 1]
```diff
@@ -72,4 +72,7 @@ This section includes the additional information for a given transaction. Inform
 - submitTimeLocal
 - submitTimeUTC
 - taxExempt
-- transactionStatus
\ No newline at end of file
+- transactionStatus
+
+{:.bs-callout-info}
+If the transaction details are not available or outdated click the **Fetch** button in menu bar to update them.
```
Comment (leslietilling) [inline src/sales/transactions.md:78]:
```suggestion
If the transaction details are not available or are outdated, click **Fetch** in the button bar to update them.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Thanks for fix. Just one minor edit for note.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**